packages/calcite-components/src/components/accordion/accordion.scss

```diff
@@ -1,41 +1,77 @@
+/**
+ * CSS Custom Properties
+ *
+ * @prop --calcite-accordion-space: defines space around and between accordion item elements. Other spacing is calculated at a +4.
```

Comment:
Do we need to expose this? If it's truly between Accordion Items (not something we currently support), should it be `gap`? The `is calculated at a +4` is a little confusing.

Reply:
`gap` is a different style prop. Since it applies to margin and padding I think the right naming convention is `-space`. However I think it does make sense to be explicit that the calculation is based off a semantic token.

Reply:
Name makes sense, can it be `-internal-` though? Doesn't seem like we'd want this to change (at least for now?)

Reply:
done.